### DISCUSSION TITLE
[TOS violation report] anti endogenic system users on crabber

### DISCUSSION THREAD

**Discussion by conner-kent** [crabber-net/crabber#185] [General]
I apologize if this is the wrong place for it I'm not sure how to report individual accounts and not tweets. This is also not meant as a call out or warning post, please do not harass anyone mentioned here. I just don't know how to privately contact people running this place. Feel free to delete this if this is entirely inappropriate for this venue. I believe these accounts break the TOS under the guidelines of no hate speech. In the least, the thread between users neil and cropspicky does violate this term.

The users I'm talking about are 'neil' 'Szymsta_' 'pipis' and 'cropspicky'

User neil has this in their description 'crabebr pretty cool szy and tari <3 endo systems and nft supporters dni' One might be mildly confused and think they're saying 'love endo systems, do not interact nft supporters'. I really doubt this as 'szy' refers to user 'Szymsta_' and tari is user cropspicky.

'Szymsta_' has this in their description "@neil and @cropspicky <3 / you can also call me szy or szymsta or apollyon / trans lesbian girl with adhd and autism / dni if supports or r an endo system please . just dont even bother" 

'pipis' has no questionable posts or description, but they do have 'Pipis (pfp by @Szymsta_ go follow)'. Either they are are the mentioned user is anti endogenic or aren't concerned by this enough to use art as their profile picture. I don't think this is concrete evidence this user is anti endogenic, but they do follow user 'neil' and 'Szymsta_' in addition to promoting the latter, who are both openly anti endogenic.

'cropspicky' has posted indicating they are anti endogenic.
Thread
tari
@cropspicky
someone kept liking my tweets and they have "dni: anti-endo" just get on the blocklist
11:40 AM · Dec 3, 2021 · Crabber Web App
2 Likes [note both by users neil and Szymsta_]
mr clang @neil · Dec 3 the downside of crabber (apparently) being used mainly by systems before is that theres a buncha endos

tari @cropspicky · Dec 3 endo systems are just singlets that wanna be special
https://github.com/crabber-net/crabber/discussions/new

Screencaps if descriptions or posts are deleted.
![cropspicky](https://user-images.githubusercontent.com/75038244/145688749-1260f377-75a0-4b13-b60f-f8732c68a4bf.png)
![scar1](https://user-images.githubusercontent.com/75038244/145688750-9a3300dd-3e78-45b1-b5a2-57e96f54b7a8.png)
![mr neil 2](https://user-images.githubusercontent.com/75038244/145688751-5fb600b4-2367-4854-80ff-65f207947f82.png)
![mr neil 1](https://user-images.githubusercontent.com/75038244/145688752-0f512ed6-89c7-4a93-9a65-a4a1e828147d.png)
![cropspicky](https://user-images.githubusercontent.com/75038244/145688749-1260f377-75a0-4b13-b60f-f8732c68a4bf.png)

**Comment by jakeledoux** [comment]
Thanks for reporting. I'll see if I can get a moderator with more knowledge on the subject to weigh on this.

**Reply by conner-kent** [reply]
Ah, thank you very much! I really appreciate the quick response!

**Comment by hqnna** [comment]
Hello, I'm one of the developers and a moderator. My general stance is people can be anti or whatever they feel like **as long as** they are not:
- harassing other users
- forcing their opinions/views on others
- or, repeatedly targeting a user

In general it's best to block if someone isn't posting something you like, but if they are harassing another user, i.e. invalidating them in replies, insulting them, etc. Report the post and block, so the team can see it and handle it.

**Reply by conner-kent** [reply]
True, that's understandable. Randomly banning just based off byf / dni standards is a rather harsh stance. Thank you for taking a look at this and responding. I appreciate the transparency and feel relieved there is some reasonable leeway to coexisting on this here webbed site, despite differing of individual stances.

Also, sorry to bring this up on a weekend, of all times. I didn't check the day, haha. I hope your future week and rest of this year goes smoothly!
